SECURITY.md

```diff
@@ -0,0 +1,49 @@
+# Security Implication of the Typeshed Stub Uploader
```

Comment:
A SECURITY.md should by convention include a way to privately report security vulnerabilities. For typing-extensions (https://github.com/python/typing_extensions/blob/main/SECURITY.md), I redirected reports to GitHub's private reporting feature, but for this repo I don't have permissions to turn on that feature.

So I'd suggest:

- Someone with the right permissions (@ilevkivskyi?) should go to https://github.com/typeshed-internal/stub_uploader/security and turn on "Private vulnerability reporting"
- In this PR we should add a line saying that any vulnerabilities should be reported to https://github.com/typeshed-internal/stub_uploader/security

We should also add a SECURITY.md to the typeshed repo pointing here.

Reply:
OK, I can do this later today, or @JukkaL should be able to do this.

Reply:
I've added such a section to the document for now, mostly taking the wording from typing-extension.

Reply:
OK, I enabled "Private vulnerability reporting" for this repo.